crates/interledger-stream/src/packet.rs

```diff
@@ -961,23 +1031,87 @@ mod fuzzing {
     }
 
     #[test]
-    #[ignore]
-    fn fuzzed_3() {
+    #[cfg(features = "strict")]
+    fn fuzzed_3_frame_content_length_prefix_should_not_have_extra_bytes() {
         #[rustfmt::skip]
         let input: &[u8] = &[
             // Version, packet type, sequence and prepare amount
             1, 14, 1, 14, 1, 14,
             // num frames
             1, 1,
-            // frame data (frame type + content)
-            1, 14, 1, 0, 1, 1, 4, 0, 255, 255, 255, 255, 255, 128, 255, 128
+            // frame data (frame type, ConnectionCloseFrame)
+            1,
+            // frame data length
+            14,
+            // ErrorCode
+            1,
+            // Content length is zero here
+            0,
+            // Here are all the extra bytes that
+            // frame data length included but not in content
+            1, 1, 4, 0, 255, 255, 255, 255, 255, 128, 255, 128
             ];
 
         // roundtrip result
         // [1, 14, 1, 14, 1, 14, 1, 1, 1, 2, 1, 0]
         //                                ^ ----^
         //                                Suspect due to content prefix length
-        roundtrip(input);
+        let b = BytesMut::from(input);
+        let pkt = StreamPacket::from_decrypted(b);
+
+        assert_eq!(
+            "Invalid Packet: Incorrect number of frames or unable to parse all frames",
+            format!("{}", pkt.unwrap_err())
+        );
+    }
+
+    #[test]
+    fn fuzzed_4_handles_unknown_error_code() {
+        #[rustfmt::skip]
+        roundtrip(&[
+            // Version, packet type, sequence and prepare amount
+            1, 14, 1, 0, 1, 14,
+            // num frames
+            1, 1,
+            // frame type
+            1,
+            // frame data
+            2,
+            // errorcode
+            89,
+            // content
+            0,
+        ]);
+    }
+
+    #[test]
+    #[cfg(feature = "roundtrip-only")]
+    fn fuzzed_5_saturating_read_var_uint_replacement_cannot_roundtrip() {
+        #[rustfmt::skip]
+        let input: &[u8] = &[
+            // Version, packet type, sequence and prepare amount
+            1, 14, 1, 0, 1, 14,
+            // num frames
+            1, 1,
+            // frame type - 0x13
+            19,
+            // frame data length
+            17,
+            // some frame data
+            1, 1,
+            // other frame data varuint length is 12, > 8
+            12, 1, 153, 14, 0, 0, 0, 58, 0, 0, 91, 24, 0,
+            // other frame data
+            1, 0
+        ];
+
+        let b = BytesMut::from(input);
+        let pkt = StreamPacket::from_decrypted(b);
+
+        assert_eq!(
+            "Invalid Packet: Incorrect number of frames or unable to parse all frames",
+            format!("{}", pkt.unwrap_err())
```

Comment:
The same as https://github.com/interledger-rs/interledger-rs/pull/719/files#r626632082
```suggestion
            &pkt.unwrap_err().to_string()
```